modules/migration-mtc-release-notes-1-8-3.adoc

```diff
@@ -0,0 +1,66 @@
+
+.Support added for {oadp-short} 1.3
+
+{mtc-short} 1.8.3 adds support to {oadp-first} as a dependency of {mtc-short} 1.8.z. link:https://issues.redhat.com/browse/MIG-1520[(MIG-1520)]
+
+[id="resolved-issues-1-8-3_{context}"]
+== Resolved issues
+
+This release has the following major resolved issues:
+
+.Restic backup does not work properly when the source workload is not quiesced
+
+The HTML file is not migrated when migrating an application with a route. The Restic backup does not function as expected when the source workload is unquiesced.
+
+This issue has been resolved in {mtc-short} 1.8.3
+
+For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2242064[Bug 2242064]
+
+.Migration controller fails to install due to unsupported value error in Velero
+
+`MigrationController` fails to install due to an unsupported value error in Velero. Updating {oadp-short} 1.3.0 to {oadp-short} 1.3.1 resolves this problem.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2267018[*BZ#2267018*])
+
+This issue has been resolved in {mtc-short} 1.8.3
+
+// Most MTC issues are in Bugzilla. Jira does not show any MTC issues.
+For a complete list of all resolved issues, see the list of link:https://issues.redhat.com/issues/?filter=12432429[{mtc-short} 1.8.3 resolved issues] in Jira.
+
+[id="known-issues-1-8-3_{context}"]
+== Known issues
+
+{mtc-full} ({mtc-short}) has the following known issues:
+
+.Not currently possible to migrate associated scc for service account in {ocp-name} 4.12
+
+It is not possible to migrate associated security context constraints (SCCs) for service account in {OCP} ({ocp-name}) version 4.12. This issue is planned to be resolved in {mtc-short} 1.8.4. link:https://issues.redhat.com/browse/MIG-1454(MIG-1454).
+
+.Vulnerability exposes confidential data stored in cookies
+
+A flaw was found in Axios 1.5.1 that exposes a confidential session token. This flaw allows a remote hacker to bypass security measures and view sensitive data.
+
+For more information, see link:https://access.redhat.com/security/cve/CVE-2023-45857[CVE-2023-45857].
+
+.Flaw in Golang `protobuf` module causes `unmarshal` function to enter infinite loop
+
+A flaw was found in Golang's `protobuf` module, where the `unmarshal` function can enter an infinite loop when processing certain invalid inputs. This issue occurs during the following situations:
+
+* Unmarshaling into a message that includes a `google.protobuf.Any` 
+* When the `UnmarshalOptions.DiscardUnknown` option is enabled. 
+
+This flaw allows an attacker to craft malicious input tailored to trigger the identified flaw in the `unmarshal` function. By providing carefully constructed invalid inputs, they could potentially cause the function to enter an infinite loop, resulting in a denial of service condition or other unintended behaviors in the affected system. 
+
+For more information, see link:https://access.redhat.com/security/cve/CVE-2024-24786[CVE-2024-24786].
```

Comment:
```suggestion
```
Bugzilla tickets moved to `Verified`. Moving to resolved issues, see [comment](https://github.com/openshift/openshift-docs/pull/73356/files#r1562562181).